airbyte-webapp/src/components/ui/InfoBox/InfoBox.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
 }
 
 const Box = styled.div`
-  background: ${({ theme }) => theme.darkBeigeColor};
```

Comment:
:information_source: This component is used for "you still need to verify your email address" and for deleted connections:

![screenshot-20221102-100110](https://user-images.githubusercontent.com/877229/199554027-555a025a-50ea-4d51-b916-c7e17195157c.png)